includes/definitions/dasan-nos.yaml

```diff
@@ -0,0 +1,10 @@
+os: dasan-nos
+text: 'Dasan NOS'
+type: network
+icon: dasan
+over:
+    - { graph: device_bits, text: 'Device Traffic' }
+discovery:
+    method: sysObjectId
+    sysObjectId:
+        - .1.3.6.1.4.1.6296.1.2.5.12
```

Comment:
zynos

quanta

powerconnect

macosx

To be honest, I want to keep the yaml simple along with detection. Maybe it's just worth sticking with matching only on an array of sysDescr or sysObjectId?